content/courses/intro-to-ada/chapters/records.rst

```diff
@@ -92,11 +97,128 @@ called component selection:
           Year  : Integer range 1 .. 3000 := 2032;
        end record;
 
+       procedure Display_Date (D : Date) is
+       begin
+          Put_Line ("Day:" & Integer'Image (D.Day)
+                    & ", Month: " & Month_Type'Image (D.Month)
+                    & ", Year:" & Integer'Image (D.Year));
+       end Display_Date;
+
        Some_Day : Date := (1, January, 2000);
 
     begin
+       Display_Date (Some_Day);
+
+       Put_Line ("Changing year...");
        Some_Day.Year := 2001;
-       Put_Line ("Day:" & Integer'Image (Some_Day.Day)
-                 & ", Month: " & Month_Type'Image (Some_Day.Month)
-                 & ", Year:" & Integer'Image (Some_Day.Year));
+
+       Display_Date (Some_Day);
     end Record_Selection;
+
+As you can see in this example, we can use the dot notation in the expression
+:ada:`D.Year` or :ada:`Some_Day.Year` to access the information stored in that
+component, as well as to modify this information in assignments. To be more
+specific, when we use :ada:`D.Year` in the call to :ada:`Put_Line`, we're
+retrieving the information stored in that component. When we write
+:ada:`Some_Day.Year := 2001`, we're overwriting the information that was
+previously stored in the :ada:`Year` component of :ada:`Some_Day`.
+
+.. _Record_Comp_Renaming:
+
+Renaming
+--------
+
+In previous chapters, we've discussed :ref:`subprogram <Subprogram_Renaming>`
+and :ref:`package <Package_Renaming>` renaming. We can rename record
+components as well. Instead of writing the full component selection using the
+dot notation, we can declare an alias that allows us to access the same
+component. This is useful to simplify the implementation of a subprogram, for
+example.
+
+We can rename record components by using the :ada:`renames` keyword in a
+varible declaration. For example:
+
+.. code-block:: ada
+
+    Some_Day : Date
+    Y        : Integer renames Some_Day.Year;
+
+Here, :ada:`Y` is an alias, so that every time we using :ada:`Y`, we are really
+using the :ada:`Year` component of :ada:`Some_Day`.
+
+Let's look at a complete example:
+
+.. code:: ada run_button project=Courses.Intro_To_Ada.Arrays.Record_Component_Renaming
+    :class: ada-run
+
+    package Dates is
+
+       type Month_Type is
```

Comment:
Let's use a better name than *_Type. How about Month_Names, or Months_In_Year, or ...  Anything but that.